ucm2/Allwinner/A64/PinePhone/PinePhone.conf

```diff
@@ -0,0 +1,65 @@
+Syntax 2
+
+# https://wiki.pine64.org/index.php/PinePhone
+# https://files.pine64.org/doc/PinePhone/PinePhone%20v1.2%20Released%20Schematic.pdf
+# https://xnux.eu/devices/feature/audio-pp.html
+
+SectionUseCase."HiFi" {
+	File "/Allwinner/A64/PinePhone/HiFi.conf"
+	Comment "Play HiFi quality music"
+}
+
+SectionUseCase."Voice Call" {
+	File "/Allwinner/A64/PinePhone/VoiceCall.conf"
+	Comment "Make a phone call"
+}
+
+FixedBootSequence [
+	# Switch playback off.
```

Comment:
I don't think that this initialization scheme is quite correct. The verb EnableSequence or global SectionDefaults sequence should disable the controlled devices  - https://www.alsa-project.org/alsa-doc/alsa-lib/group__ucm__conf.html - to put the hardware to a good initial state before any device from the verb is used. Move the "turn mixer paths off" commands to SectionDefaults for multiple verbs. It will be executed before any verb is activated.

The other controls may be moved to the standard BootSequence, too. But if they are not supposed (or dangerous) to be modified by user later, you can keep them here. Note that alsactl should restore the previous state of all ALSA driver controls on boot.